corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -348,3 +361,33 @@ <h4 class="panel-title accordion-toggle">
     {% include 'reports/partials/data_corrections_modal.html' %}
   {% endif %}
 {% endif %}
+
+<div class="modal fade" id="delete-form-modal">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label='{% trans_html_attr "Close" %}'><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">{% trans 'Permanently Delete Form' %}</h4>
+      </div>
+      <div class="modal-body">
+        <form action="{% url "delete_form" domain instance.form_id %}" method="POST">{% csrf_token %}
+          <p class="alert alert-warning">
+            <i class="fa fa-exclamation-triangle"></i>
+            {% blocktrans %}
+              It is important you read the entire message below thoroughly before completing this action.
+            {% endblocktrans %}
+          </p>
+          <p>Once deleted, you will not be able to recover this form. Are you sure you want to delete this form?</p>
```

Comment:
translate

Reply:
Thank you e197797aa2bf0d94f0e19d0c6d567179371ddd77